plugins/sudo/sudo.plugin.sh

```diff
@@ -0,0 +1,14 @@
+#! bash oh-my-bash.module
+
+add_sudo() {
+  local prefix suffix
+  prefix="${READLINE_LINE:0:$READLINE_POINT}"
+  suffix="${READLINE_LINE:$READLINE_POINT}"
```

Comment:
By the way, do we need to split `READLINE_LINE` into `prefix` and `suffix`?

Reply:
What do you think?

Reply:
removed prefix and suffix in last commit